tests/test.ts

```diff
@@ -99,7 +99,7 @@ test("init and account fetch", async () => {
   expect(
     res.i128Field.eq(new BN("-85070591730234615865843651857942052874"))
   ).toBe(true)
-  expect(res.bytesField).toStrictEqual([1, 2, 255, 254])
+  expect(res.bytesField).toEqual(Buffer.from([1, 2, 255, 254]))
```

Comment:
Could we use `Uint8Array.from(...)` in this filed instead of Buffer? I know Buffer is a subclass of Uint8Array but it's a bit confusing.

Reply:
yes

Reply:
The bytes field is actually a buffer, and the comparison fails if the right hand side is not a buffer either.

Reply:
Hmm, does the borsh layout decode return a Buffer instead of Uint8Array in that case?

Reply:
Yeah borsh seems to be pretty much Buffer based.